internal/builders/generic/README.md

```diff
@@ -370,19 +370,12 @@ jobs:
     steps:
       [...]
       - name: Build using bazel
-        # =================================================
-        #
-        # Step 2: Add an `id: build` field
-        #         to your build step.
-        #
-        # =================================================
-        id: build
         run: |
           # Your normal build workflow targets here
           bazel build //path/to/target_binary //path/to_another/binary
           # ======================================================
           #
-          # Step 3: Copy the binaries from `bazel-bin` path (i.e.,
+          # Step 2: Copy the binaries from `bazel-bin` path (i.e.,
```

Comment:
nit: do we really need to number these?

Reply:
Wanted to order the steps. Shall we remove them?

Reply:
I'm ok with keeping them if you think it's helpful.